### PR TITLE
Add `helm lint` CI job. Split tag/release for the Helm chart from the entire repo with tags: `helm/X.Y.Z-b1`

### PR DESCRIPTION
**Pull request by andrewrothstein**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,6 @@ unused_deps
 buildozer
 java_formatter.jar
 lcov
+
+# emacs undo-changes
+**/*.~undo-tree~
```
Comment (werkt) [inline .gitignore:26]: Intended for this PR?

File: README.md [hunk 2]
```diff
@@ -138,9 +138,11 @@ buildfarm_images()
 To install with helm:
 
 ```bash
+# https://github.com/andrewrothstein/bazel-buildfarm/releases/download/helm%2F0.2.2/buildfarm-0.2.2.tgz
```
Comment (werkt) [inline README.md:141]: Intended comment referencing your repo?

File: kubernetes/helm-charts/buildfarm/values.yaml [hunk 3]
```diff
@@ -131,22 +131,25 @@ shardWorker:
...
+  #extraVolumes:
+  # - name: additionalSecret
+  #   secret:
+  #   secretName: my-secret
+  #   defaultMode: 0600
+
+  #extraVolumeMounts:
+  # - name: customConfig
+  #   mountPath: /mnt/config
+  #   readOnly: true
```
Comment (werkt) [inline kubernetes/helm-charts/buildfarm/values.yaml:147]: This looks similar to your other PR #1604, is this extra/intended, and will the template resolution treat it kindly to have these fields missing?

File: README.md (hunk 2 above)
Comment (andrewrothstein) [reply README.md]: fixed

**Review by werkt — APPROVED** [review]
Thanks for updates, landing

**Pull request merged**